src/token/fedauth-info-parser.js

```diff
@@ -0,0 +1,62 @@
+const async = require('async');
+
+var readFedAuthInfoOpt = (data, infoData, callback) => {
```

Comment:
What do you think about just changing this to a regular function definition?

```js
function readFedAuthInfoOpt(data, infoData, callback) {
  ...
}
```

Arrow functions are usually only preferred when we want to have a function bound to the surrounding `this` scope, or we want a shorter way of defining a closure, neither of which applies here. 😅